README.ja.md

```
@@ -193,6 +193,75 @@ use BEAR\ToolUse\Attribute\Tool;
+
+### DIモジュールでのバインド
+
+```php
+$this->bind(ConfirmationHandlerInterface::class)->to(CliConfirmationHandler::class);
+```
+
+### 動作の仕組み
+
+LLMのテキストレスポンスが確認メッセージとして使われます。テンプレートは不要です。
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)